src/main/java/steps/MainPageSteps.java

```diff
@@ -42,11 +45,14 @@ public static void goToCart() {
         validateURL("https://automationteststore.com/index.php?rt=checkout/cart");
     }
 
-    public static void goToCheckout() {
-        var goToCheckout = getWebDriverInstance().findElement(By.xpath("//*[@id='cart_checkout2']"));
-        goToCheckout.click();
+    public static void updateCart() {
```

Comment:
Oh...sorry those should be in the CartPageSteps class not here:P

Reply:
Done! Im waiting for approval! :D